libp2p/Host.cpp

```diff
@@ -24,6 +24,7 @@
 #include <chrono>
 #include <thread>
 #include <mutex>
+#include <boost/asio.hpp>
```

Comment:
By targeted, do you mean a more platform specific target define? Will look into this later tonight.